### PR TITLE
Vizceral troubleshooting README.md

### PR DESCRIPTION
**Pull request by bantonsson**
This should maybe go into the normal Cinnamon doc?

### CODE REVIEW

File: cinnamon/vizceral/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+cinnamonsamplevzc_web_1
+cinnamonsamplevzc_backend_1
+cinnamonsamplevzc_elasticsearch_1
+```
+
+If any of the containers are not running, then CTRL-C the `docker-compose command` and make sure that all containers are stopped and removed.
+
+```
+> docker ps -a --format "{{.Names}}" | grep cinnamonsamplevzc | xargs docker stop
+> docker ps -a --format "{{.Names}}" | grep cinnamonsamplevzc | xargs docker rm
```
Comment (henrikengstrom) [inline cinnamon/vizceral/README.md:18]: Maybe add a `-f` to the end of the `docker rm` command?

**Comment by henrikengstrom** [comment]
LGTM.

Yes, this can definitely be added to the Cinnamon doc.

Perhaps it is worthwhile adding Kibana to the whole setup from the start?

**Comment by bantonsson** [comment]
Yes, I think adding Kibana as a default would make things easier.

File: cinnamon/vizceral/README.md (hunk 1 above)
Comment (bantonsson) [reply cinnamon/vizceral/README.md:18]: Force all the things...

Comment (henrikengstrom) [reply cinnamon/vizceral/README.md:18]: Use the force as someone said :)

**Comment by bantonsson** [comment]
Adding this to main Cinnamon documentation instead.

**Pull request closed without merging**